Testing/Exec/TranEval/GPU_misc.H

```diff
@@ -37,11 +37,20 @@ initialize_data(
     L[n] = phi[n] - plo[n];
     P[n] = L[n] / 4.0;
   }
+#ifdef USE_MANIFOLD_EOS
```

Comment:
Are these ifdefs going to be all over our app codes do you think?

Reply:
None should be required for PeleC because it won't use this capability. They will pop up in a few places in LMeX, but mostly just case data initialization similar to here. For the most part, adding an EOS function that computed Rho from the RhoYs removes the need for ifdefs.